jobserver/views/job_requests.py

```diff
@@ -174,6 +176,8 @@ def get_form_kwargs(self):
         return super().get_form_kwargs() | {
             "actions": [a["name"] for a in self.actions],
             "backends": backends_to_choices(self.backends),
+            "database_actions": self.database_actions,
```

Comment:
Actually, it needs the output of `load_pipeline(self.project)`, which isn't passed around on `self`. I thought it might be able to use `self.actions`, but that's lost the actual action obj that has the `is_database_action` method.  I think I'd need to make `data` in the dispatch `self.data`, which would then mean it has to go into that slightly icky except block